scripts/nb-tester/qiskit_docs_notebook_tester/config.py

```diff
@@ -74,7 +74,7 @@ def get_notebook_jobs(args: argparse.Namespace) -> Iterator[NotebookJob]:
 
             patch = config.get_patch_for_group(group)
 
-            if patch:
+            if patch and not "# nb-tester: allow-write" in patch:
```

Comment:
New minor feature: We can allow writing notebooks to disk with specific patches, see the new README entry for more information.